src/pages-helpers/pupil/lessons-pages/getProps.ts

```diff
@@ -95,22 +95,15 @@ export const getProps = ({
           })
         : resolveOakHref({ page: "pupil-year-index" });
 
-    const { transcriptSentences, hasWorksheet } = await (async () => {
-      try {
-        const { transcriptSentences, hasWorksheet } =
-          await requestLessonResources({
-            lessonContent: content,
-          });
-
-        return { transcriptSentences, hasWorksheet };
-      } catch (e) {
-        if (page === "preview") {
-          return { transcriptSentences: [], hasWorksheet: false };
-        } else {
-          throw e;
-        }
+    const transcriptSentences = await requestLessonResources({
```

Comment:
we no longer need to calculate hasWorksheet as we get it in the lessonContent query